src/templates/home-page.js

```diff
@@ -39,10 +41,20 @@ export const HomePageTemplate = ({
   }, [])
 
   const [liquidityRewardsAPYs, isFetching] = useLiquidityRewardsAPY()
-  const highestAPY =
-    liquidityRewardsAPYs.length > 0
-      ? `${liquidityRewardsAPYs[0].value}% APY.`
-      : "loading..."
+
+  const renderHighestAPY = () => {
+    return liquidityRewardsAPYs.length > 0 ? (
+      liquidityRewardsAPYs[0].value.toString().length > 10 ? (
```

Comment:
It's not needed anymore

Initially I added because SlideInAnimation doesn't work for more than 10 characters (and by doesn't work i meant the characters stayed hidden on top and didn't slide down) but i fixed it. The probability of the APY (without fractional part) having more than 10 digits is also very low (if even possible).

If there will be more than 10 character the Slide In Animation will just show the number/text without sliding it down. We can always increase max number of chars by adding more `@include`s in `slide-in-animation.scss`

Reply:
Fixed in https://github.com/keep-network/website/pull/453/commits/e9d5f8fc0365bf217309b9f4c9450377802421ad